### PR TITLE
Added tan, sinh and cosh to keras backend

### PR DESCRIPTION
**Pull request by F20170604**
Functions Added:
1) tan
2) Hyperbolic sin
3) Hyperbolic cos

### CODE REVIEW

**Review by kyscg — APPROVED** [review]
Thanks for your contribution @Ayush517 . Changes LGTM !!!

**Comment by Dref360** [comment]
This is not part of the Keras API.
If you want to make it part of the Keras API, please submit a PR on https://github.com/keras-team/keras/.

Also, this PR requires new unit tests.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by qlzh727** [comment]
For TF high level API, we don't expect to add new API in keras/backend, which also is just a wrapper around the TF API. The keras backend API is there for historical reason, eg when we copy them from keras-team/keras to TF. If you feel that this API is useful in keras (for multi-backend purpose), please create a PR in keras-team/keras.

Thanks.

**Comment by qlzh727** [comment]
As pre comment, I am going to close this PR now. Feel free to create PR on keras-team/keras if you feel it is necessary.
